packages/medusa/src/api/routes/admin/products/index.ts

```diff
@@ -211,7 +211,9 @@ export type AdminProductsDeleteRes = {
  *   products:
  *     type: array
  *     items:
- *       $ref: "#/components/schemas/Product"
+ *       oneOf:
+ *         - $ref: "#/components/schemas/Product"
+ *         - $ref: "#/components/schemas/PricedProduct"
```

Comment:
Now matches the typed response.
https://github.com/medusajs/medusa/blob/4424301ae27ea0dbcd97bb284462e56ce2a52698/packages/medusa/src/api/routes/admin/products/index.ts#L227-L229